### PR TITLE
feat(classification): add data type classification data to db.go

### PR DESCRIPTION
**Pull request by elsapet**
## Description
* Add data types and related data type patterns to default database
* Refactor db.Default() usage since it's now more than just recipes

## Checklist

- [ ] I've added test coverage that shows my fix or feature works as expected.
- [ ] I've updated or added documentation if required.
- [ ] I've included usage information in the description if CLI behavior was updated or added.

### CODE REVIEW

File: pkg/classification/db/db.go [hunk 1]
```diff
@@ -28,7 +46,44 @@ type RecipeType string
...
+func Default() DefaultDB {
+	return DefaultDB{
+		Recipes:                        defaultRecipes(),
+		DataTypes:                      defaultDataTypes(),
+		DataTypeClassificationPatterns: defaultDataTypeClassificationPatterns(),
+		KnownPersonObjectPatterns:      defaultKnownPersonObjectPatterns(),
+	}
+}
+
+func defaultRecipes() []Recipe {
```
Comment (elsapet) [inline pkg/classification/db/db.go:85]: I'm not sure if there's a golang way to extract the code shared between these `defaultX()` methods? It's all the same except for the types 🙈

File: pkg/classification/db/db.go [hunk 2]
```diff
@@ -43,27 +98,99 @@ func Default() []Recipe {
...
 	if err != nil {
-		log.Fatalf("failed to unmarshal db %e", err)
-		return nil, err
+		log.Fatalln(err)
+	}
+
+	for _, file := range files {
+		val, err := knownPersonObjectPatternsDir.ReadFile("known_person_object_patterns/" + file.Name())
+		if err != nil {
+			fmt.Println(err)
```
Comment (cfabianski) [inline pkg/classification/db/db.go]: Should probably be `log.Errorf`

File: pkg/classification/db/db.go [hunk 3]
```diff
@@ -43,27 +98,99 @@ func Default() []Recipe {
...
+
+	files, err := dataTypeClassificationPatternsDir.ReadDir("data_type_classification_patterns")
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	for _, file := range files {
+		val, err := dataTypeClassificationPatternsDir.ReadFile("data_type_classification_patterns/" + file.Name())
+		if err != nil {
+			fmt.Println(err)
```
Comment (cfabianski) [inline pkg/classification/db/db.go]: Should probably be `log.Errorf`

**Review by cfabianski — COMMENTED** [review]
Let's be careful with the logging. 
Looks good otherwise.

**Review by cfabianski — APPROVED** [review]

File: pkg/classification/db/db.go (hunk 1 above)
Comment (elsapet) [reply pkg/classification/db/db.go:85]: Resolving this comment so I can merge but still open for advice on improving this

**Pull request merged**